src/_assets/css/_code_shared.scss

```diff
@@ -1,4 +1,14 @@
-@import 'vendor/code-prettify/_prettify';
+
+// TODO this should be moved from this file for better 
```

Comment:
I'm confused as to why there is a todo on commented out code.

Reply:
Also, I'd prefer the todo to reference an open issue so this will get done.

Reply:
Actually: https://dart-lang.github.io/linter/lints/flutter_style_todos.html

Reply:
btobin must have left this here as a reference for when the rearch was integrated into site-www. I'll remove the comment and make the necessary changes downstream in site-www when integrating these changes.